package.json

```diff
@@ -46,14 +46,14 @@
     "null-loader": "3.0.0",
     "postcss": "7.0.16",
     "postcss-custom-properties": "6.3.1",
-    "react": "15.6.2",
-    "react-addons-test-utils": "15.6.2",
+    "react": "16.13.1",
```

Comment:
So this means we can also remove the `lifecycleCompatibilityPolyfill` which is only used [here](https://github.com/schrodinger/fixed-data-table-2/blob/master/src/FixedDataTableCell.js#L349) to support `getDerivedStateFromProps`?